lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
This brings up an interesting point about the remedation points algorithm as well: per-instance issues means that a new instance of the same duplication ends up creating a new instance on our end, but does make existing issues worse.